lesson_07/conditionals/src/lesson7.ts

```diff
@@ -85,18 +128,21 @@ export function binarySearch(
   end: number,
   value: number,
 ): number {
+  const pivotIndex = Math.floor((start + end) / 2); // The index in the middle of the array.
+
   if (end < start) {
     // The range is not valid so just return -1.
     return -1;
+  } else if (values[pivotIndex] == value) return pivotIndex;
```

Comment:
Remember, you should use `===` in typescript

Reply:
will do, thank you.